src/Microsoft.Security.Utilities.Core/SecretMasker.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.Security.Utilities;
 /// <returns>The escaped or encoded literal.</returns>
 public delegate string LiteralEncoder(string literal);
 
-public class SecretMasker : ISecretMasker
+public sealed class SecretMasker : ISecretMasker
 {
     private readonly IRegexEngine _regexEngine;
```

Comment:
>private [](http://example.com/codeflow?start=4&length=7)

we've got privates, both readonly and not sprinkled throughout this class, can we just group them somewhere? SEV4

Reply:
Yes, I agree and I'll do it. I was being overly conservative in trying to minimize the diff, but also strongly wanted to do this!